src/react_native/core.cljs

```diff
@@ -21,6 +21,7 @@
 (def text (reagent/adapt-react-class (.-Text ^js react-native)))
 (def text-input (reagent/adapt-react-class (.-TextInput ^js react-native)))
 
+(def pressable (reagent/adapt-react-class (.-Pressable ^js react-native)))
```

Comment:
I think it's a good time for us to reconsider our usages of `TouchableHighlight`, `TouchableOpacity` and `TouchableWithoutFeedback` and use the new `Pressable` API.

Also, I've been thinking about talking to designers about micro-interactions for buttons and pressable "things". Might not even be hard depending on what they come up with and could bring a nice touch to the user experience.